app/backend/src/couchers/servicers/resources.py

```diff
@@ -1,16 +1,21 @@
 import logging
-from pathlib import Path
-
-import grpc
 
+from couchers.resources import get_language_dict, get_region_dict, get_terms_of_service
 from pb import resources_pb2, resources_pb2_grpc
 
 logger = logging.getLogger(__name__)
 
-resources_folder = Path(__file__).parent / ".." / ".." / ".." / "resources"
-
 
 class Resources(resources_pb2_grpc.ResourcesServicer):
     def GetTermsOfService(self, request, context):
-        with open(resources_folder / "terms_of_service.md", "r") as f:
-            return resources_pb2.GetTermsOfServiceRes(terms_of_service=f.read())
+        return resources_pb2.GetTermsOfServiceRes(terms_of_service=get_terms_of_service())
+
+    def GetRegions(self, request, context):
+        return resources_pb2.GetRegionsRes(
+            regions=[resources_pb2.Region(alpha3=alpha3, name=name) for alpha3, name in get_region_dict().items()]
```

Comment:
Throw in a sorted() as well to get a deterministic order, not depending on Python's randomized hash function.